.github/workflows/performance-benchmarks-bayesmark.yml

```diff
@@ -0,0 +1,112 @@
+      pruner-list:
+        description: 'Pruner List: A list of pruners to check the performance. Should be a whitespace-separated list of Optuna pruners. Each pruner must exist under `optuna.pruners`.'
+        required: false
+        default: 'NopPruner'
+      pruner-kwargs-list:
+        description: 'Pruner Arguments List: A list of pruner keyword arguments. Should be a whitespace-separated list of json format dictionaries.'
+        required: false
+        default: '{}'
+      budget:
+        description: 'Number of Trials'
+        required: false
+        default: '80'
+      repeat:
+        description: 'Number of Studies'
+        required: false
+        default: '10'
+      plot-warmup:
+        description: Include warm-up steps in plots.
+        type: boolean
+        default: true
+
+
+jobs:
+  benchmarks:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        dataset: [breast, digits, iris, wine, diabetes]
+        model: [kNN, SVM, DT, RF, MLP-sgd, ada, linear]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Python3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install Python libralies
+      run: |
+        python -m pip install --upgrade pip
+        pip install --progress-bar off -U setuptools wheel
+
+        pip install --progress-bar off .
+        python -c 'import optuna'
+        optuna --version
+
+        pip install --progress-bar off bayesmark
+        pip install --progress-bar off matplotlib
+
+    - name: Run performance benchmark
+      run: |
+        python benchmarks/run_bayesmark.py \
+          --dataset ${{ matrix.dataset }} \
+          --model ${{ matrix.model }} \
+          --budget ${{ github.event.inputs.budget }} \
+          --repeat ${{ github.event.inputs.repeat }} \
+          --sampler-list '${{ github.event.inputs.sampler-list }}' \
+          --sampler-kwargs-list '${{ github.event.inputs.sampler-kwargs-list }}' \
+          --pruner-list '${{ github.event.inputs.pruner-list }}' \
+          --pruner-kwargs-list '${{ github.event.inputs.pruner-kwargs-list }}' \
+          --plot-warmup '${{ github.event.inputs.plot-warmup }}'
+    
+    - uses: actions/upload-artifact@v2
+      with:
+        name: benchmark-plots
+        path: plots
+    
+    - uses: actions/upload-artifact@v2
+      with:
+        name: partial-reports
+        path: partial
+
+  report:
+    needs: benchmarks
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - uses: actions/download-artifact@v2
+      with:
+        name: partial-reports
+        path: partial
+    
+    - name: Run benchmark report builder
+      run: |
+        python benchmarks/report_bayesmark.py
+    
+    - uses: actions/upload-artifact@v2
+      with:
+        name: benchmark-report
+        path: report
+    
+    - uses: geekyeggo/delete-artifact@v1
+      with:
+        name: partial-reports
```

Comment:
Missing newline. 🍵

Reply:
Fixed in 3db2f4fd83b44532018fa4adfd05eaff0f35a77d. There was also extra line between sections, so I removed that. I wonder if it would be worth introducing some kind of linter for those workflow files...